skins/oasis/js/Tracking.js

```diff
@@ -748,7 +749,53 @@ jQuery(function ($) {
 			if (label !== undefined) {
 				track({
 					browserEvent: event,
-					category: 'recent-wiki-activity',
+					category: 'forum-activity-module',
+					label: label
+				});
+			}
+		});
+
+		// TODO: refactor while working on https://wikia-inc.atlassian.net/browse/XW-3202
+		/** related-threads-module **/
+		$wikiaRail.find('#ForumRelatedThreadsModule').on('mousedown', 'a', function (event) {
+			var label,
+				el = $(event.target);
+
+			// Primary mouse button only
+			if (event.which !== 1) {
+				return;
+			}
+
+			if (el.hasClass('more')) {
+				label = 'activity-more';
+			} else if (el.closest('.edited-by').length > 0) {
+				label = 'activity-username';
+			} else if (el.closest('em').length > 0) {
```

Comment:
Could we add some class to em element? Relying on tag name may be very unsure, and moreover with class added we would be more consistent.